contracts/token/Minter.sol

```diff
@@ -95,14 +110,35 @@ contract Minter is Manager, IMinter {
      * @dev Set the reward token amounts for the round. Only callable by the RoundsManager
      */
     function setCurrentRewardTokens() external onlyRoundsManager whenSystemNotPaused returns (bool) {
+        setInflation();
+
         currentMintableTokens = mintedTokensPerRound();
         currentMintedTokens = 0;
-        currentRedistributableTokens = redistributableTokensPerRound();
+        currentRedistributableTokens = redistributionPool;
```

Comment:
I know that redistributing in the current round might be simplest, but do you think it would be wrong just to spread it over some number `n` rounds, rather than all at once? It could arbitrarily be 100 rounds?

Reply:
Picking an arbitrary number of rounds feels a bit weird, but I also don't like redistributing the entire pool in the next round since its an immediate benefit for any transcoder that chose to distribute fees after its delegators unbonded in the previous round. The latter might be worse in terms of how transcoder incentives are affected so maybe spreading it out even if it is over an arbitrary number of rounds is better at least for the time being